src/Model/Contact.php

```diff
@@ -1260,10 +1260,10 @@ public static function getPostsFromUrl($contact_url, $thread_mode = false, $upda
 	/**
 	 * Returns posts from a given contact id
 	 *
-	 * @param integer $cid
+	 * @param integer $cid         Contact ID
 	 * @param bool    $thread_mode
-	 * @param integer $update
-	 * @param int     $parent
+	 * @param integer $update      Update mode 
```

Comment:
Please use the more ubiquitous `int`, or at least the same type string in the same doc block.